### PR TITLE
Bump jquery to fix select2 problem

### PR DESCRIPTION
**Pull request by strangrjrjr**
### What github issue is this PR for, if any?
Resolves local development issues

### What changed, and why?
Bumped jquery and select2 packages

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

**Comment by littleforest** [comment]
@strangrjrjr can you rebase this branch with master? Or start fresh and cherry-pick over your new commit(s)? You have a bunch of unnecessary commits in here.

File: app/models/user.rb [hunk 1]
```diff
@@ -5,6 +5,9 @@ class User < ApplicationRecord
   has_paper_trail
   devise :database_authenticatable, :invitable, :recoverable, :rememberable, :validatable
 
+  validates :email, presence: true
```
Comment (jmkoni) [inline app/models/user.rb:8]: Looks good, but this looks unrelated to "bumping jquery to fix select2". Is there another issue this is solving?

Comment (strangrjrjr) [reply app/models/user.rb:8]: Hmm, odd. These are commits from #726; this PR was for the bump in `package.json` and `yarn.lock`

**Comment by strangrjrjr** [comment]
Ah, I'll rebase. Sorry about that!

**Comment by strangrjrjr** [comment]
Closing this PR; I'll give it a fresh shot.
